src/Juvix/Prelude/Pretty.hs

```diff
@@ -123,6 +126,9 @@ toAnsiText useColors
 toPlainText :: (HasTextBackend a) => a -> Text
 toPlainText = Text.renderStrict . toTextStream
 
+toPlainTextTrim :: (HasTextBackend a) => a -> Text
+toPlainTextTrim = Text.renderStrict . rTW . toTextDoc
```

Comment:
why not rendering to text and then removing the trailing whitespace as we discussed?